postprocessing/plevel_interpolation/compile_plev_interpolation.sh

```diff
@@ -5,6 +5,9 @@ cd ./exec
 
 source $GFDL_BASE/src/extra/env/$GFDL_ENV
 
-../bin/mkmf -p plev.x -t ../bin/mkmf.template.ia64 -c "-Duse_netCDF" -a ../src ../src/path_names ../src/shared/mpp/include ../src/shared/include
+compiler=${GFDL_MKMF_TEMPLATE:-ia64}
```

Comment:
Do we want to add any other templates to the folder or is the idea people can add them if they want? Just wondering if we have other templates sitting on different branches that could come for free in the PR.